bin/o2a-run-sys-test

```diff
@@ -826,6 +837,17 @@ elif [[ "${PHASE}" == "test-oozie" ]]; then
     fetch_dataproc_environment_info
     test_oozie
     download_test_artifacts "oozie"
+elif [[ "${PHASE}" == "compare-artifacts" ]]; then
```

Comment:
nit: Maybe it could be sth like "test-compare-artifacts"? To make it clear that running tests via both methods is also included in this phase.

Reply:
I updated the PR.